rhel/SPECS/grafana.spec

```diff
@@ -15,16 +15,16 @@
 %endif
 
 Name:           percona-%{repo}
-Version:        6.5.1
-Release:        4%{?dist}
+Version:        6.7.3
+Release:        1%{?dist}
 Summary:        Grafana is an open source, feature rich metrics dashboard and graph editor
 License:        ASL 2.0
 URL:            https://%{import_path}
 Source0:        https://%{import_path}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
-Source2:        grafana-node_modules-v6.5.1.el7.tar.gz
-Source4:        percona-favicon.ico
-Patch0:         grafana-5.4.2-change-icon.patch
-Patch1:         grafana-6.3.6-share-panel.patch
+#Source2:        grafana-node_modules-v6.7.3.el7.tar.gz
```

Comment:
Not sure we need to keep it around, since there is a `yarn install` further down.

Reply:
Removed this source from rpm spec